distribution/docs/src/main/jdocs/content/_developingComponents/developing-token-validators-contents.adoc

```diff
@@ -0,0 +1,13 @@
+:title: Developing Token Validators
+:type: developingComponent
+:status: published
+:link: _developing_token_validators
+:summary: Creating a custom token validator.
+:order: 12
+
+Token validators are used by the Security Token Service (STS) to validate incoming token requests.
+The `TokenValidator` CXF interface must be implemented by any custom token validator class.
+The `canHandleToken` and `validateToken` methods must be overridden.
+The `canHandleToken` method should return true or false based on the `ValueType` value of the token that the validator is associated with.
+The validator may be able to handle any number of different tokens that you specify.
+The `validateToken` method returns a `TokenValidatorResponse` object that contains the `Principal` of the identity being validated and also validates the `ReceivedToken` object that was collected from the RST (`RequestSecurityToken`) message.
```

Comment:
✏️  "that was collected from the RST (`RequestSecurityToken`)" --> "collected from the `RequestSecurityToken` (RST)"

Reply:
updated.